manifests/w/Wesang/Yogiyo/4.0.6/Wesang.Yogiyo.installer.yaml

```diff
@@ -0,0 +1,19 @@
+# Created using wingetcreate 1.6.5.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.6.0.schema.json
+
+PackageIdentifier: Wesang.Yogiyo
+PackageVersion: 4.0.6
+InstallerType: exe
+InstallModes:
+- silent
+InstallerSwitches:
+  Silent: -q
+Installers:
+- Architecture: x86
+  InstallerUrl: https://apps.rpsyogiyo.io/win/go/4.0.6/GoYogiyo_Light_Installer_4.0.6.exe
+  InstallerSha256: A0B8E037A7D12E4D8A53FED7BB04E03A258EF662A558CD8BDE8E49D97A98B71E
+  AppsAndFeaturesEntries:
+  - DisplayName: GoYogiyo
```

Comment:
```suggestion
```

`Installers > AppsAndFeaturesEntries > DisplayName` is exactly the same as `PackageName`, I think it should be removed.